nebula/ui/components/VPNServerList.qml

```diff
@@ -62,77 +129,306 @@ FocusScope {
+                    right: parent.right
+                }
+
+                VPNCollapsibleCard {
+                    anchors.horizontalCenter: parent.horizontalCenter
+
+                    iconSrc: "qrc:/ui/resources/tip.svg"
+                    contentItem: VPNTextBlock {
+                        text: VPNl18n.ServersViewRecommendedCardBody
+                        textFormat: Text.StyledText
+                        Layout.fillWidth: true
+                    }
+                    title: VPNl18n.ServersViewRecommendedCardTitle
+                    width: parent.width - VPNTheme.theme.windowMargin * 2
+                }
+
+                // Status component
+                // TODO: Refresh server list and handle states
+                VPNClickableRow {
+                    id: statusComponent
+
+                    anchors {
+                        leftMargin: VPNTheme.theme.windowMargin * 0.5
+                        rightMargin: VPNTheme.theme.windowMargin * 0.5
+                    }
+                    canGrowVertical: true
+                    height: statusTitle.implicitHeight + VPNTheme.theme.vSpacingSmall
+                    rowShouldBeDisabled: !(VPNController.state === VPNController.StateOff)
+
+                    onClicked: {
+                        focusScope.updateServerData();
+                    }
+
+                    RowLayout {
+                        anchors {
+                            fill: parent
+                            leftMargin: VPNTheme.theme.listSpacing
+                            rightMargin: VPNTheme.theme.listSpacing
+                        }
+                        spacing: VPNTheme.theme.listSpacing
+
+                        VPNIcon {
+                            source: "qrc:/nebula/resources/clock.svg"
+
+                            Layout.alignment: Qt.AlignLeft | Qt.AlignTop
+                            Layout.topMargin: VPNTheme.theme.listSpacing
+                        }
+
+                        VPNInterLabel {
+                            id: statusTitle
+
+                            Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
+                            Layout.fillWidth: true
+                            Layout.preferredWidth: parent.width
+                            Layout.maximumWidth: parent.width
+
+                            color: VPNTheme.theme.fontColor
+                            horizontalAlignment: Text.AlignLeft
+                            // TODO: Replace string arg with the actual formatted date
+                            text: !statusComponent.rowShouldBeDisabled
+                                ? VPNl18n.ServersViewRecommendedTextStatus.arg("5m")
+                                : VPNl18n.ServersViewRecommendedTextStatusWithInfo.arg("5m")
+                            wrapMode: Text.WordWrap
+                        }
+
+                        Item {
+                            Accessible.name: VPNl18n.ServersViewRecommendedTextInfo
```

Comment:
I don't see this in strings.yaml? And should the `Accessible.name` field be attached to the `ClickableRow`?

Reply:
Dang — changed the string id and moved it to the `ClickableRow`.